test/documents_test.dart

```diff
@@ -37,6 +52,31 @@ void main() {
       expect(doc?['title'], equals('The Hobbit 2'));
     });
 
+    test('Update documents in batches', () async {
+      const batchSize = 10;
```

Comment:
This is an excellent recommendation I like to share whenever is possible: http://blog.plataformatec.com.br/2014/04/improve-your-test-readability-using-the-xunit-structure/

By using this pattern your test structure will look way more readable!

Reply:
Thanks for this tip, definitely will use it from now on!